esmvaltool/diag_scripts/droughtindex/collect_drought_func.py

```diff
@@ -441,30 +455,21 @@ def get_latlon_index(coords, lim1, lim2):
 
 def plot_map_spei_multi(cfg, data_dict, input_filenames, colormap='jet'):
     """Plot contour maps for multi model mean."""
-    print("data_dict['data']")
-    print(data_dict['data'])
-    # mask = np.isnan(data_dict['data'])
     spei = np.ma.array(data_dict['data'], mask=np.isnan(data_dict['data']))
-    # np.ma.masked_less_equal(spei, 0)
-    print("spei")
-    print(spei)
 
     # Get latitudes and longitudes from cube
-    # lats = data_dict['latitude']
     lons = data_dict['longitude']
-    lons = np.where(lons > 180, lons - 360, lons)
-    # sort the array
-    index = np.argsort(lons)
-    lons = lons[index]
-    data_dict.update({'longitude': lons})
-    # mesh_ind = np.ix_(range(360), index)
-    spei = spei[np.ix_(range(len(data_dict['latitude'])), index)]
+    if max(lons) > 180.0:
+        lons = np.where(lons > 180, lons - 360, lons)
+        # sort the array
+        index = np.argsort(lons)
+        lons = lons[index]
+        spei = spei[np.ix_(range(len(data_dict['latitude'])), index)]
```

Comment:
```suggestion
        spei = spei[np.ix_(range(lons.size), index)]
```

Reply:
Thanks @valeriupredoi for the fast check! Yes, that line is sorting the data to fit to the change in lon.

Reply:
lats was not used (to have not to many local variables). So it is probably a good idea to use size instead of len, but the variable has to remain data_dict['latitude'].

Reply:
Maybe using `enumerate` can help in here. In any case, how ready is this for 2.6?

Reply:
@lukruh did the recipe finish successfully then? @valeriupredoi does this change address your comments?

Reply:
excellent! My apologies for leaving this in the bushes yesterday (it was too hot to work), and happy to see all is working fine! Cheers for merging @sloosvel :beer: